### PR TITLE
Fix 404-ing costume in "Text" sprite

### PR DESCRIPTION
**Pull request by Kenny2github**
### Resolves

Solves #466

### Proposed Changes

Replaced 404-ing link to 3.0 asset with 2.0 asset

### Reason for Changes

The entire "Text" sprite was un-addable because of one 404-ing costume.

### CODE REVIEW

**Comment by paulkaplan** [comment]
Hmmm it seems that the SVG has some quirk that isn't allowing the thumbnail to be rendered correctly. @rschamp has dealt with this issue several times before, he can probably help with this. We can't use this as-is because this is what it looks like as a thumbnail (note that it is fine on the stage, just the thumbnail is messed up)
image

**Comment by Kenny2github** [comment]
@paulkaplan Okie dokie, once @rschamp looks into it I'll make the relevant changes - then this problem should be completely fixed 😄

**Comment by Kenny2github** [comment]
@paulkaplan any updates?

File: src/lib/libraries/sprites.json [hunk 1]
```diff
@@ -14328,7 +14328,7 @@
                 {
                     "costumeName": "text keep scratching",
                     "baseLayerID": -1,
-                    "baseLayerMD5": "22a916248a294eadc07a7a065f5154e8.svg",
+                    "baseLayerMD5": "b3de9a77e2c1bb2f3e29f29a03d869ea.svg",
```
Comment (rschamp) [inline src/lib/libraries/sprites.json]: Please update to `97c9e6ef78d4f1987fd8c6e5042963cb.svg` and it will start working 🌈

Comment (Kenny2github) [inline src/lib/libraries/sprites.json]: Will do!

Comment (Kenny2github) [inline src/lib/libraries/sprites.json]: Done!

**Pull request merged**